Include/internal/pycore_interp.h

```diff
@@ -168,8 +171,25 @@ struct _is {
 
     struct ast_state ast;
     struct type_cache type_cache;
+
+    struct {
+        // See _PyRuntimeState._preallocated for an explanation.
+        PyThreadState tstate;
+        // XXX Pre-allocate as many objects from above as possible here.
+    } _preallocated;
 };
 
+#define _PyInterpreterState_INIT \
```

Comment:
This is only used once, please remove it.

Reply:
FYI, I brought this back to avoid so much nesting and clutter in `_PyRuntimeState_INIT`.